app/controllers/admin/genres_controller.rb

```diff
@@ -0,0 +1,21 @@
+class Admin::GenresController < Admin::BaseController
+  def new
```

Comment:
this shouldn't have been necessary since the form itself would have been part of the non-admin Genre#index page.